### PR TITLE
fix: implemented workaround for Uno.Toolkit.UI

### PR DESCRIPTION
**Pull request by juliecantin**
GitHub Issue: #

## Proposed Changes

Bug fix

## What is the current behavior?

Adding a reference to AsyncWebView in a Uno 5 project causes the following error:
Conflicting values for resource 'Files/Uno.UI.Toolkit/Themes/Generic.xbf'

## What is the new behavior?
Adding a reference to AsyncWebView in a Uno 5 project causes no compilation error.

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: global.json
Comment (Soap-141) [inline global.json:1]: Why are we adding this?

Comment (juliecantin) [reply global.json:1]: To fix compilation issues, as per @carlh98 's advice

Comment (Soap-141) [reply global.json:1]: Is the pipeline having the same issue? If not we could remove it and only use it locally if needed?

Comment (carlh98) [reply global.json:1]: yeah the issue is in the pipeline since net 6 is out of support for mobile

**Review by Soap-141 — APPROVED** [review]

**Comment by carlh98** [comment]
Closing this, since fix implemented with https://github.com/nventive/AsyncWebView/pull/42

**Pull request closed without merging**